radiomics/firstorder.py

```diff
@@ -283,10 +284,11 @@ def getUniformityFeatureValue(self):
     where a greater uniformity implies a greater heterogeneity or a
     greater range of discrete intensity values.
     """
+    eps = numpy.spacing(1)
 
     bins = imageoperations.getHistogram(self.binWidth, self.targetVoxelArray)[0]
     try:
-      bins = bins / (float(bins.sum()))
+      bins = bins / (float(bins.sum() + eps))
```

Comment:
Both approaches have merit in my opion. I propose to leave this like it is for now and address the issue in a next hangout call. I'd like to handle these situations in one way throughout the toolbox.